src/span.js

```diff
@@ -107,6 +156,7 @@ export default class Span {
      * @return {[type]} [description]
      */
     addTags(keyValuePairs) {
```

Comment:
nit: should we clarify that `keyValuePairs` is actually a hash (maybe use a different name?), and not an array of things?